Update Actions and Notification commands ahead of iOS 2024.7 release (#1081)
home-assistant/companion.home-assistant: docs/core/actions.md

> ## Using Actions

Reviewer: ```suggestion
## Using actions
```

> After having fulfilled all Action data (Text, name etc), tap "Create automation" button available on this same screen.

Reviewer: ```suggestion
After having filled in all action data (text, name, etc.), tap the **Create automation** button.
```

## Home Screen Quick Actions

> [Home Screen Quick Actions](https://support.apple.com/guide/iphone/keep-apps-handy-iph414564dba/ios#iph1ffcbd691) provides a convenient shortcut to your actions and is accessed by press and holding the Home Assistant companion app icon on your home screen.

Reviewer: ```suggestion
[Home Screen Quick Actions](https://support.apple.com/guide/iphone/keep-apps-handy-iph414564dba/ios#iph1ffcbd691) provides a convenient shortcut to your actions. To access it, press and hold the Home Assistant companion app icon on your home screen.
```
